src/utils/crypto.util.ts

```diff
@@ -1,16 +1,35 @@
+import scrypt from 'react-native-scrypt';
 import {
   secureCellSealWithPassphraseDecrypt64,
   secureCellSealWithPassphraseEncrypt64,
   symmetricKey64
 } from 'react-native-themis';
 import { from, Observable, of } from 'rxjs';
-import { catchError } from 'rxjs/operators';
+import { map, switchMap, catchError } from 'rxjs/operators';
 
 export interface EncryptedData {
   encrypted64: string;
   salt64: string;
 }
 
+const scrypt$ = async (password: string, salt: string) =>
+  from<string>(await scrypt(password, salt, 65536, 8, 1, 32, 'base64'));
```

Comment:
number of rounds looks good, however, i urge you to test it on different devices, as the processing might be too slow, especially on androids. if it's too slow, try decreasing rounds to 32768 or even 16384.

see https://cryptobook.nakov.com/mac-and-key-derivation/scrypt